assignment7.txt

```diff
@@ -0,0 +1,14 @@
+Jack Rosen
+1. ++*p increments the value at point p. *p++ increments p and then returns the value. *++p increments p and then returns the value as well.
+2. It is only guaranteed for certain operators. The first level of operators goes from left to right, the second goes from right to left, and the rest go from left to right.
+3.The advantages of using pointers is that they are very powerful and help a person work with arrays. It is also a way to access a variable without having to call for it.
```

Comment:
Pointers are more efficient in handling arrays and data tables and can be used to return multiple values from a function via function arguments. They also allow references to functions and C to support dynamic memory management. Lastly, Pointers reduce length and complexity of programs!